content/en/docs/collector/transforming-telemetry.md

```diff
@@ -23,38 +23,35 @@ a significant impact on collector performance.
 **Processor**:
 [filter processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor)
 
-The filter processor allows users to filter telemetry based on `include` or
-`exclude` rules. Include rules are used for defining "allow lists" where
-anything that does _not_ match include rules is dropped from the collector.
-Exclude rules are used for defining "deny lists" where telemetry that matches
-rules is dropped from the collector.
+The filter processor allows users to filter telemetry using
+[OTTL](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/ottl/README.md).
```

Comment:
You'll want to add `OTTL` to the cspell section at the top of the doc

Reply:
don't we have OTTL in cspell yet? 😮